src/app/globals.css

```
@@ -11,8 +11,8 @@
 
   /* Pretendard */
   --font-pretendard:
-    "Pretendard", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial,
-    "Apple SD Gothic Neo", "Noto Sans KR", sans-serif;
+    "Pretendard", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.2.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)